pandas/core/dtypes/common.py

```diff
@@ -320,8 +324,9 @@ def is_datetimetz(arr):
     True
     """
 
-    # TODO: do we need this function?
-    # It seems like a repeat of is_datetime64tz_dtype.
+    warnings.warn("'is_datetimetz' is deprecated and will be removed in a "
+                  "future version.  Use 'is_datetime64tz_dtype' instead.",
+                  FutureWarning, stacklevel=2)
 
     return ((isinstance(arr, ABCDatetimeIndex) and
```

Comment:
change the impl then to directly call its replacement

Reply:
done